src/prompt/files_prompt.rs

```diff
@@ -134,10 +134,13 @@ impl<'a> FilesPrompt<'a> {
 
             let y_offset = buffer.lines() + self.focused_index;
 
-            let focused_color = style("").with(ct::Color::Blue).to_string();
-            let modified_color = style("").with(ct::Color::Rgb { r: 96, g: 112, b: 218 }).to_string();
-            let untracked_color = style("").with(ct::Color::Rgb { r: 96, g: 218, b: 177}).to_string();
-            let deleted_color = style("").with(ct::Color::Rgb { r: 218, g: 96, b: 118}).to_string();
+            let focused_color = ct::Color::Blue;
+            let default_color = ct::Color::White;
+
+            let status_untracked = style('+').with(ct::Color::Rgb { r: 96, g: 218, b: 177 });
+            let status_modified = style('/').with(ct::Color::Rgb { r: 96, g: 112, b: 218 });
```

Comment:
Maybe `*` for modified. You often get a `*` or `●` kind of icon in an editor tab when a file is modified.

Also, some kind of arrow for moved/renamed like `⇾` or `➟`

Reply:
I don't think we can't actually display moved/renamed files in any capacity.

The issue is that if you rename a file, Git will automatically stage it. Glint, with staged files, will skip the file selection prompt entirely (as I fixed that for renamed files the other day).

If you were to use `git reset` to unstage the renaming/moving, Git will actually separate that into two files -- a deleted and an untracked. It doesn't keep its renamed status at that point and will break the link between the two.

That would be difficult functionality to add in, as you'd have to check every combination of removed and untracked files to see if they're alike enough to justify a "Renamed" label. I'm sure that would come with false positives too.